o1vm/src/mips/constraints.rs

```diff
@@ -44,14 +44,13 @@ impl<Fp: Field> InterpreterEnv for Env<Fp> {
     /// variable/input of our circuit.
     type Position = MIPSColumn;
 
-    // Use one of the available columns. It won't
-    // create a new column every time this function is called. The number
-    // of columns is defined upfront by crate::mips::witness::SCRATCH_SIZE.
+    // Use one of the available columns. It won't create a new column every time
+    // this function is called. The number of columns is defined upfront by
+    // crate::mips::witness::SCRATCH_SIZE.
     fn alloc_scratch(&mut self) -> Self::Position {
         // All columns are implemented using a simple index, and a name is given
-        // to the index.
-        // See crate::SCRATCH_SIZE for the maximum number of columns the circuit
-        // can use.
+        // to the index. See crate::SCRATCH_SIZE for the maximum number of
```

Comment:
```suggestion
        // to the index. See `crate::SCRATCH_SIZE` for the maximum number of
```

Reply:
Only to check that the doc is up-to-date when building it.

Reply:
Ah. Just noticed that it is only in documentation starting with /// and it has to be between square brackets.